src/modules/item/utils.ts

```diff
@@ -242,3 +243,8 @@ export function getCategories(contents: Record<string, any> | undefined = {}) {
 export function isComplexFile(fileName: string) {
   return fileName.endsWith('.gltf') || fileName.endsWith('.glb')
 }
+
+export function isItemPNGFile(fileName: string) {
+  const name = fileName.toLowerCase()
+  return name.indexOf(THUMBNAIL_PATH) === -1 && !name.includes('_mask') && fileName.endsWith('.png')
+}
```

Comment:
I don't know about this. The method is called `isItemPNGFile` but it receives a filename and it's checking for a `_mask`. There's also already a `isPNGModel` in `CreateItemModal` but that one is not checking for _mask (just png's like the name implies).

It seems to me that maybe we could just extract the original method on `CreateItemModal` and use some of the methods there?

```diff
const modelPath = fileNames.find(
-   fileName => isComplexFile(fileName) || (fileName.indexOf(THUMBNAIL_PATH) === -1 && fileName.endsWith('png'))
+   fileName => this.isModelPath(filename)
)
```

```ts
  isModelPath(fileName: string) {
    fileName = fileName.toLowerCase()
    const isMask = fileName.includes('_mask')
    return isComplexFile(fileName) || (fileName.indexOf(THUMBNAIL_PATH) === -1 && !isMask && this.isPNGModel(filename))
  }
```

what do you think?
(coded here, not ment to copypasta)